WeakAuras.lua

```diff
@@ -3138,9 +2980,268 @@ function WeakAuras.RegisterTriggerSystem(types, triggerSystem)
   tinsert(triggerSystems, triggerSystem);
 end
 
+function WeakAuras.GetTriggerStateForTrigger(id, triggernum)
+  triggerState[id][triggernum] = triggerState[id][triggernum] or {}
+  return triggerState[id][triggernum];
+end
+
+local function startStopTimers(id, cloneId, triggernum, state)
```

Comment:
Here are the functions that take a triggerState and evaluate them. They are called from UpdatedTriggerState below.

This specific function starts or stops auto hide timers.